integr8ly/keycloak-operator
# Intly 1896

## verification
- Firstly, just have an OS3 and an OS4 cluster to hand... ha!

Luckily, I  have 2 clusters you can use:
os3: 
- url: https://master.pbrookes-1639.openshiftworkshop.com/console

os4: 
- web: http://console-openshift-console.apps.cluster-913d.913d.openshiftworkshop.com
- oc: https://api.cluster-913d.913d.openshiftworkshop.com:6443

On each cluster, do the following: 
- create a test namespace
- create a keycloak CR
- create a keycloakRealm CR
- run the operator (the new image is at quay.io/integreatly/keycloak-operator:v1.3.5)
- check that SSO is provisioned and the realm is reconciled.
